### ISSUE TITLE
Suggestion: visualize that a message was deleted in the server

### ISSUE THREAD

**Issue by rogerluan** [SharonBrizinov/slack-anti-delete#5]
Hi! 👋 

This looks like a super fun project! I appreciate the technical details you documented in here 😊 

I was wondering if this can be productized a little bit, by displaying to the user (somehow) that a given message was deleted, while keeping it there (visible, readable). Based on your research so far, would you say this is possible (and/or easily achievable)?

It could be something like changing the background color of the message to red, or displaying a different icon/badge, or something else, idk. Just food for thought 😄 

Congrats on the initiative! 🚀

**Comment by SharonBrizinov** [comment]
Yeah, it's definitely possible. I was thinking about a strike through or a specific badge for deleted messages and something similar for edited messages. It's also not that complicated, you'll need to track the edit message flow, understand how the edit message objects are built and handled, and finally reuse it wherever you want (delete message flow, edit flow, etc). 

Hopefully someone from the community will take up the glove 😊

**Comment by JARMourato** [comment]
I hear @rogerluan is up for the task 💪🏻

**Comment by rogerluan** [comment]
hahahaha @JARMourato you know I'd love to. Maybe I'll give it a shot in a free weekend 😁 won't promise anything though 🙈
